generator/wrappers.py

```diff
@@ -131,6 +131,7 @@ def wrap_algo(algo, ver):
     'algorithms::kdtree_knn_classification': ['seed',], # deprecated
     'algorithms::lasso_regression::training': ['optionalArgument'], # internal stuff
     'algorithms::lasso_regression::prediction': ['algorithms::linear_model::interceptFlag',], # parameter
+    'algorithms::multi_class_classifier': ['algorithms::multi_class_classifier::getTwoClassClassifierModels',] # unsupported return type ModelPtr*
```

Comment:
Will this continue working? https://github.com/IntelPython/daal4py/blob/master/daal4py/sklearn/svm/svm.py#L229

Reply:
yes